src/editor/parts.ts

```diff
@@ -364,6 +373,9 @@ class RoomPillPart extends PillPart {
     get className() {
         return "mx_RoomPill mx_Pill";
     }
+
+    // FIXME: We do this to shut up the linter, is there a way to do this properly
+    onClick = undefined;
```

Comment:
Is there any way?

Reply:
well the issue is you say its abstract which means it must be set on all children, see suggestion above